src/keywords/absolutePath.js

```diff
@@ -0,0 +1,56 @@
+function errorMessage(schema, data, message) {
+  return {
+    keyword: 'absolutePath',
+    params: { absolutePath: data },
+    message,
+    parentSchema: schema,
+  };
+}
+
+function getErrorFor(shouldBeAbsolute, data, schema) {
+  const message = shouldBeAbsolute
+    ? `The provided value ${JSON.stringify(data)} is not an absolute path!`
+    : `A relative path is expected. However, the provided value ${JSON.stringify(
+        data
+      )} is an absolute path!`;
+
+  return errorMessage(schema, data, message);
+}
+
+export default (ajv) =>
+  ajv.addKeyword('absolutePath', {
+    errors: true,
+    type: 'string',
+    compile(expected, schema) {
+      function callback(data) {
+        let passes = true;
+        const isExclamationMarkPresent = data.includes('!');
+        const isCorrectAbsoluteOrRelativePath =
+          expected === /^(?:[A-Za-z]:\\|\/)/.test(data);
```

Comment:
A recent webpack PR adds windows network shares as absolute path starting with `\\`

Reply:
https://github.com/webpack/webpack/pull/9382? Looks not merged right now

Reply:
tests are missing, but `\\computer\share` is an absolute path for sure.

It's difficult to test inside of webpack, but should be easy to test here.

Reply:
:+1: so i just move this here and test